modules/core/src/main/java/org/apache/ignite/internal/processors/cache/CacheConfigurationSplitterImpl.java

```diff
@@ -39,16 +41,24 @@
     /** Near cache configuration to hold default field values. */
     private static final NearCacheConfiguration<?, ?> DEFAULT_NEAR_CACHE_CONFIG = new NearCacheConfiguration<>();
 
+    /** Fields set for V1 splitter protocol. */
+    private static final String[] FIELDS_V1 = {"evictPlcFactory", "evictFilter", "storeFactory", "storeSesLsnrs"};
```

Comment:
It seems that `nearEvictPlcFactory` is missing

Reply:
I can not find where is it used.
I will try to remove it and run in TC.